docs/diagrams/AddTaskSequenceDiagram.puml

```diff
@@ -5,7 +5,7 @@ box Logic LOGIC_COLOR_T1
 participant ":LogicManager" as LogicManager LOGIC_COLOR
 participant ":InTrackParser" as InTrackParser LOGIC_COLOR
 participant ":AddTaskCommandParser" as AddTaskCommandParser LOGIC_COLOR
-participant "d:AddTaskCommand" as AddTaskCommand LOGIC_COLOR
+participant "at:AddTaskCommand" as AddTaskCommand LOGIC_COLOR
```

Comment:
Don't forget to update the .png file too, if not the developer guide won't reflect these changes

Reply:
Noted, thank you for the suggestion.